src/video/uikit/SDL_uikitappdelegate.m

```diff
@@ -51,12 +51,7 @@ int SDL_RunApp(int argc, char *argv[], SDL_main_func mainFunction, void *reserve
      */
```

Comment:
There's a comment above about memory which isn't necessarily bad advice, but is no longer relevant.